tests/safeds/ml/nn/layers/test_forward_layer.py

```diff
@@ -177,3 +177,49 @@ def test_should_assert_that_different_forward_layers_have_different_hash(
 )
 def test_should_assert_that_layer_size_is_greater_than_normal_object(layer: ForwardLayer) -> None:
     assert sys.getsizeof(layer) > sys.getsizeof(object())
+
+
+@pytest.mark.parametrize(
+    ("activation_function", "expected_activation_function"),
+    [
+        ("sigmoid", nn.Sigmoid),
+        ("relu", nn.ReLU),
+        ("softmax", nn.Softmax),
+        ("none", None),
+    ],
+    ids=["sigmoid", "relu", "softmax", "none"],
+)
+def test_should_set_activation_function(activation_function: str, expected_activation_function: type | None) -> None:
+    forward_layer = ForwardLayer(neuron_count=1, activation_function=activation_function)
+    forward_layer._input_size = 1
+    internal_layer = forward_layer._get_internal_layer(
+        activation_function="relu",
+    )
+    # check if the type gets overwritten by constructor
+    assert (
+        internal_layer._fn is None
+        if expected_activation_function is None
+        else isinstance(internal_layer._fn, expected_activation_function)
+    )
+
+
+@pytest.mark.parametrize(
+    "activation_function",
+    [
+        "a",
+        "b",
+        "c",
+        "d",
+    ],
+    ids=["sigmoid", "relu", "softmax", "none"],
```

Comment:
Reduce to one unknown string, there is no need to test this four times. Put something like "unknown" as id

Reply:
added